Makefile

```diff
@@ -4,7 +4,9 @@ build:
 	docker build -t score-service:latest .
 
 run:
-	docker run -p ${APP_PORT}:${APP_PORT} -it score-service:latest
+	$(eval include .env)
```

Comment:
what's this for?

Reply:
ah, after removing -p this isn't needed anymore